### ISSUE TITLE
Update Arm toolchains to 6.3.0

### ISSUE THREAD

**Issue by gustavsl** [plaidml/toolchain#8]
The x86 toolchain was updated to 6.3.0 [in this commit](https://github.com/plaidml/toolchain/commit/43e1bc29c2871f1a524937895686ae6c26ab2bc1), but the Arm toolchains were not updated.

This causes [this issue](https://github.com/plaidml/plaidml/issues/248) when trying to compile for Arm.

It's not just a matter of updating [CROSSTOOL](https://github.com/plaidml/toolchain/blob/master/CROSSTOOL) since [workspace.bzl](https://github.com/plaidml/toolchain/blob/master/workspace.bzl) fetches the actual pre-compiled toolchain from another URL.

Any chance of updating the Arm toolchain URLs or sharing the crosstool-ng configs so I can compile this myself?

Thanks!

**Comment by flaub** [comment]
I don't have the crosstool-ng configs handy but I'll go digging for them at some point and share them here.

**Comment by flaub** [comment]
OK, here's the config I used for crosstool-ng with the earlier compiler. You can use this as a starting point, but you'll still have to update to a c++14 compiler that also uses the oldest libc you can get away with.

https://gist.github.com/flaub/eb97297b2733d9d279a671b09ae40142

**Comment by gustavsl** [comment]
@flaub Thanks! Apparently plaidml builds without issues now.

I've imported your config, upgraded it since I'm using crosstool-ng 1.24.0 (`ct-ng upgradeconfig`), updated glibc to 2.23, gcc to 6.5.0 and isl to 1.6.1. isl was a little tricky because the mirror is still down, so I had to manually fetch the tarball from wayback machine. [This is the new config](https://gist.github.com/gustavsl/831d4292eddaf5d7aab63692e5279035).
The toolchain built just fine, I've uploaded the tarball to my fork so I could link it on workspace.bzl and built the plaidml and plaidml-keras wheels.
